src/components/Button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from './Button.module.css';
+
+const button = props => {
+  const { name, width, color } = props;
+
+  const styles = {
+    width,
+    backgroundColor: color,
+  };
+  return (
+    <button type="button" className={classes.Button} style={styles}>{name}</button>
+  );
+};
+
+button.propTypes = {
+  name: PropTypes.string.isRequired,
+  width: PropTypes.string,
+  color: PropTypes.string,
+};
```

Comment:
i suppose `"width"` (line 6 + line 19) is what the requirement calls `size`  [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)]

this property is supposed to be `Boolean`, not string [see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)]